src/controllers/list.js

```diff
@@ -791,21 +791,37 @@ class ItemsView {
 
                 const itemType = item ? item.Type : null;
 
-                if (itemType === 'MusicGenre' || params.type !== 'Programs' && itemType !== 'Channel') {
+                if ((itemType === 'MusicGenre' || params.type !== 'Programs' && itemType !== 'Channel')
+                    // Folder, Playlist views
+                    && itemType !== 'UserView'
+                    // Only Photo (homevideos) CollectionFolders are supported
+                    && !(itemType === 'CollectionFolder' && item?.CollectionType !== 'homevideos')
+                ) {
+                    // Hide Play All buttons
                     hideOrShowAll(view.querySelectorAll('.btnPlay'), false);
```

Comment:
`hideOrShowAll(..., false)` shows the element.
_The name makes it difficult to understand the semantics of the function._

Reply:
I agree the function name is very strange which is why I added comments here. It is probably worthy of a separate PR to rename it.

Reply:
Oh doh. I think I added the comments before I got this working and then didn't update them. Fixed now.